torchinductor/triton_ops/conv.py

```diff
@@ -138,9 +138,9 @@ def _kernel_delta_x_hwc(
             delta_xh_ptrs += BLOCK_K
             delta_xw_ptrs += BLOCK_K
             delta_xc_ptrs += BLOCK_K
-            delta_xh = tl.load(delta_xh_ptrs, mask=off_x_crs < CRS)
-            delta_xw = tl.load(delta_xw_ptrs, mask=off_x_crs < CRS)
-            delta_xc = tl.load(delta_xc_ptrs, mask=off_x_crs < CRS)
+            delta_xh = tl.load(delta_xh_ptrs, mask=off_x_crs < CRS, other=0)
```

Comment:
I'm surprised you need to set `other` for offsets - data with invalid offsets should never be loaded, it should be masked.

Reply:
I found it's weird, everthing works fine for float32. But when I only change dtype to float16, some tests would failed the correctness checking